src/main/webapp/add_event_page/add_event.html

```diff
@@ -4,43 +4,51 @@
     <meta charset="UTF-8">
     <title>Add Event</title>
     <link rel="stylesheet" href="style.css">
-    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDlz0NQ1mc6_UU6OHExGKIrpnkJ5q9BqPs"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDlz0NQ1mc6_UU6OHExGKIrpnkJ5q9BqPs&libraries=places"></script>
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
     <script src="script.js"></script>
   </head>
-  <body onload="initMap()">
+  <body>
     <div id="header">
       <h1>Add New Event</h1>
     </div>
-
-    <div id="form">
-        <form action="/new_event.php">
+        <form id="form">
             <label for="name">Event Name:</label>
-            <input type="text" id="name" name="name"><br>
-            <label for="desc">Description:</label>
-            <textarea id="desc" name="desc"></textarea><br>
-            <label for="loc">Building Name:</label>
-            <input type="text" id="loc" name="loc"><br>
-            <label for="date">Date:</label>
-            <input type="text" id="date" name="date"><br>
+            <input type="text" id="name" name="name" required><br>
+            <label for="description">Description:</label>
+            <textarea id="description" name="description" required></textarea><br>
+            <label for="location_name">Building Name:</label>
+            <input type="text" id="location_name" name="location_name" required><br>
+            <label for="date">Date and Time:</label>
+            <input type="datetime-local" id="date" name="date" required><br>
             <label>Event Type:</label>
-            <select name="ev_type" id="ev_type">
-              <option value="Social">Study</option>
+            <select id="event_type" name="event_type" required onchange="checkType()">
+              <option value="Study">Study</option>
               <option value="Social">Social</option>
             </select><br>
-            <label>Subject:</label>
-            <select name="subject" id="subject">
+
+            <label id="subject1">Subject:</label> 
+            <select name="subject" id="subject2">
               <option value="English">English</option>
               <option value="History">History</option>
               <option value="Science">Science</option>
               <option value="Math">Math</option>
               <option value="Social">Social</option>
             </select><br>
-            <input type="submit" value="Submit"><br>
+            <button type="button" onclick="submitEvent()">Submit</button><br>
```

Comment:
This line would be
`<input type="submit" onclick="submitEvent()"><br>`
(so that the required inputs will be checked)